lib/src/lib.ts

```diff
@@ -56,6 +57,20 @@ export function isHttpMethod(method: string): method is HttpMethod {
   }
 }
 
+export function isHttpContentType(contentType: String): contentType is HttpContentType {
```

Comment:
Use `string`, not `String`. See https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
sorry it is a typo